<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GDrive App</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/Icon/Icon/Icon_main.png</normaloff>:/Icon/Icon/Icon_main.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,3,0,2">
    <item>
     <widget class="QLabel" name="label_CurrentReply">
      <property name="text">
       <string>Current reply</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTreeView" name="treeView_Reply"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Console</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAcount">
    <property name="title">
     <string>Account</string>
    </property>
    <addaction name="action_Login"/>
    <addaction name="action_Logout"/>
    <addaction name="action_Remember_me"/>
    <addaction name="action_Refresh_Token"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionUpload_File"/>
    <addaction name="actionDownload_file"/>
    <addaction name="action_Search_file_folder"/>
    <addaction name="actionGet_file_matadata"/>
    <addaction name="actionUpdate_file"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;About</string>
    </property>
    <addaction name="action_About_User"/>
   </widget>
   <addaction name="menuAcount"/>
   <addaction name="menu_About"/>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDownload_file">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Download file</string>
   </property>
  </action>
  <action name="action_Search_file_folder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Search file / folder</string>
   </property>
  </action>
  <action name="actionGet_file_matadata">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Get file matadata...</string>
   </property>
  </action>
  <action name="actionUpdate_file">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Update File</string>
   </property>
  </action>
  <action name="actionUpload_File">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Upload File...</string>
   </property>
  </action>
  <action name="action_Login">
   <property name="text">
    <string>&amp;Login</string>
   </property>
  </action>
  <action name="action_Refresh_Token">
   <property name="text">
    <string>&amp;Refresh Token</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_Logout">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Logout</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_Remember_me">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Remember me</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_About_User">
   <property name="text">
    <string>&amp;About User</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
